kornia/enhance/adjust.py

```diff
@@ -192,14 +192,14 @@ def adjust_gamma(
     The input image is expected to be in the range of [0, 1].
 
     Args:
-        input: Image to be adjusted in the shape of :math:`(*, N)`.
+        input: Image to be adjusted in the shape of :math:`(*, H, W)`.
```

Comment:
what's the final decision about shape ? I see some HxW vs CxHxW - we should state this somewhere in the documentation. /cc @shijianjian @ducha-aiki @lferraz

Reply:
IMO, `CxHxW` should be indicated when we explicitly need the channel dimension (for e.g., the color module), and `HxW` when we don't. I changed here cuz `(*, N)` doesn't really make sense for image, and users may think they need to flatten the image to use the function.